@W-16989657: Update to-deploy-using-rest-connect.adoc to inform not support for asian characters (#287)
mulesoft/docs-exchange: modules/ROOT/pages/to-deploy-using-rest-connect.adoc

* Anypoint Platform, including REST Connect, does not support API fragments for OAS.
* TLS support can be enabled only for Mule 4 (SmartConnector) connectors. This feature is not supported for Mule 3 (DevKit).

> * Chinese / Japanese characters inside api specs or its dependencies are not yet supported.

Reviewer: ```suggestion
* Chinese and Japanese characters in API specs or their dependencies are not yet supported.
```
